clusters/app.ci/supplemental-ci-images/assisted-ci-tools/temp-images-for-migration.yaml

```diff
@@ -1,5 +1,58 @@
 kind: ImageStream
 apiVersion: image.openshift.io/v1
+metadata:
+  namespace: edge-infrastructure
+  name: ocm-2.8
+spec:
+  tags:
+  - name: assisted-installer
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-installer
+      namespace: edge-infrastructure
+  - name: assisted-installer-agent
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-installer-agent
+      namespace: edge-infrastructure
+  - name: assisted-installer-controller
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-installer-controller
+      namespace: edge-infrastructure
+  - name: assisted-service
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-service
+      namespace: edge-infrastructure
+  - name: assisted-image-service
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-image-service
+      namespace: edge-infrastructure
+  - name: assisted-test-infra
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-test-infra
+      namespace: edge-infrastructure
+  - name: cluster-api-provider-agent
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:cluster-api-provider-agent
+      namespace: edge-infrastructure
+  - name: assisted-service-scripts
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-service-scripts
+      namespace: edge-infrastructure
+  - name: assisted-service-index
+    from:
+      kind: ImageStreamTag
+      name: ocm-2.7:assisted-service-index
+      namespace: edge-infrastructure
+---
```

Comment:
One more thing, we're gonna need ``assisted-test-infra-internal`` as well, because it's being used in nutanix&vsphere jobs:
https://github.com/openshift/release/blob/b96d29219b85d71197fe9676e64fc4823984921a/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml#L22-L25

https://github.com/openshift/release/blob/0896191a7c5ec03f0bbb9038bc1fe8ac14f887a2/ci-operator/step-registry/assisted/vsphere/setup/configure/assisted-vsphere-setup-configure-ref.yaml#L3